index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link href="styles.css" rel="stylesheet">
+    <title>Orange Dream Poetry Magazine</title>
+  </head>
+
+  <body>
+    <h1>Orange Dream Poetry Magazine</h1>
+
+  <h2>Poems</h2>  
+    <ul>
+      <li>Awaking in New York by Maya Angelou</li>
+      <li>Phenomenal Woman by Maya Angelou</li>
+      <li>Burnt Ships by Ibsen</li>
+      <li>The Miner by Ibsen</li>
+      <li>To The Survivors by Ibsen</li>
+      <li>A Coat by W.B. Yeats</li>
+      <li>A Dialogue of Self and Soul by W.B. Yeats</li>
+      <li>The Song of Wandering Aengus by W.B. Yeats</li>
+    </ul>
+
+  <script src="scripts.js"></script>
+  </body>
+
+
+</html>
```

Comment:
Apparently there is a programming standard that asks for an empty line at the end of a code file. However, I asked Shawn and he said it's not as critical in html as it is in Python or C++, but a good habit to get into. Whether we add a line or not, I vote for a merge of both of these files.